src/batch/src/task/task_manager.rs

```diff
@@ -180,4 +201,39 @@ mod tests {
             .to_string()
             .contains("can not create duplicate task with the same id"));
     }
+
+    #[tokio::test]
+    async fn test_task_aborted() {
+        let manager = BatchManager::new();
+        let plan = PlanFragment {
+            root: Some(PlanNode {
+                children: vec![],
+                identity: "".to_string(),
+                node_body: Some(NodeBody::GenerateSeries(GenerateSeriesNode {
+                    start: Some(make_i32_literal(1)),
+                    stop: Some(make_i32_literal(10000000)),
```

Comment:
This test is a hack to make the execution long enough so that we can abort it during execution, otherwise the task will transition successfully to `Finished` status.